cirq-core/cirq/circuits/moment.py

```diff
@@ -148,8 +148,6 @@ def operation_at(self, qubit: raw_types.Qid) -> Optional['cirq.Operation']:
         """
         if self.operates_on([qubit]):
             return self.__getitem__(qubit)
-        else:
-            return None
```

Comment:
So it turns out that mypy won't let you do that, but you can still say
```python
if ...:
    return ...
return None
```

Reply:
right! done